wordpress-mobile/AztecEditor-Android
# Make `em` the default tag for italic

### Fix
Gutenberg has `<em>` defined as the tag for italic formatting. Aztec can handle `<em>` but converts it to `<i>` when sending back the html to gutenberg-mobile. This causes inconsistencies in the UI and should be addressed.

### Test
1. Checkout `gutenberg-mobile` PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/275
1. Use the following patch to make changes to Aztec on `gutenberg-mobile`:
```
curl "https://gist.githubusercontent.com/Tug/50f0a51b7833a94d5f05f4fa7020a289/raw/3317f9ab4dd096cb2a358c44bb5e1085a8a15550/debug-with-aztec.patch" | git apply -v
```
2. Run the app and check that italic formatting is selected when the cursor/selection is on italic
formatting

### Review
@hypest
